apps/che/src/main/fabric8/deployment.yml

```diff
@@ -144,13 +144,17 @@ spec:
         imagePullPolicy: "IfNotPresent"
         name: che
         readinessProbe:
-          tcpSocket:
+          httpGet:
+            path: /api/system/state
             port: 8080
-          initialDelaySeconds: 20
+            scheme: HTTP
+          initialDelaySeconds: 60
```

Comment:
Probe checks start right after a container is started - from this point to the point when Che server is running, it usually takes ~30 seconds. Probes are executed periodically, see [documentation](https://docs.openshift.com/container-platform/3.5/dev_guide/application_health.html#container-health-checks-using-probes).